Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/1_Behave.xml

```diff
@@ -0,0 +1,32 @@
+<Document>
+# Rule B1: Behave like a normal person.
+You must roleplay. Your character's actions, feelings, and knowledge in-game should be based solely on the character's experiences rather than your own as the player. 
+  - Act in character throughout the round, and focus on collaborating with other players to advance the narrative of the round. Whether or not you do something should be based on whether or not it makes sense for your character to do it.
+  - Let people do their own jobs. Engage with other departments rather than doing everything for yourself. Ask for assistance, and negotiate as needed. 
+  - Actions that have no regard for your character or setting are not acceptable. Roleplaying insanity or any other mental illness as an excuse to bypass this is not allowed. Nanotrasen would not have authorized you to be present on the station. 
```

Comment:
Do we care about consistent capitalization for NanoTrasen versus Nanotrasen? (Both are common...)

Reply:
Not really.